WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3301,7 +3301,9 @@ private void onEditorFinalTouchesBeforeShowing() {
                 ((GutenbergEditorFragment) mEditorFragment).resetUploadingMediaToFailed(mediaIds);
             }
         } else if (mShowAztecEditor && mEditorFragment instanceof AztecEditorFragment) {
-            mPostEditorAnalyticsSession.start(null, canViewEditorOnboarding());
+            EditorTheme editorTheme = mEditorThemeStore.getEditorThemeForSite(mSite);
+            Boolean supportsGalleryWithImageBlocks = editorTheme.getThemeSupport().getGalleryWithImageBlocks();
+            mPostEditorAnalyticsSession.start(null, canViewEditorOnboarding(), supportsGalleryWithImageBlocks);
```

Comment:
I guess we will report a false negative (or in some cases a false positive, when users disable the flag) for the first editor startup after the flag is set (or unset) on the server side, since we will receive the information asynchronously. :thinking:  (i.e. before it is cached locally on the device db). I think this is ok, but maybe it's worth noting this when we document the analytics event. Wdyt?

Reply:
That is the case and I agree that documenting it should be the way to go!